docs/docsite/rst/userguide/teams.rst

```diff
@@ -118,26 +126,22 @@ To add permissions to a Team:
 3. Click to select the resource to assign team roles and click **Next**.
 
 .. image:: ../common/images/teams-permissions-templates-select.png
+   :alt: Permissions Templates Select
```

Comment:
This implies it is just for templates. I suggest something like:
"Add Teams Permissions Wizard step 2, choose the resources from the list."